acoular/__init__.py

```diff
@@ -126,3 +122,4 @@
     WriteWAV,
 )
 from .trajectory import Trajectory
+from .version import __author__, __date__, __version__

```

Comment:
why is this needed?

Reply:
It is needed to call `acoular.__version__` etc.